fastlane/Pluginfile

```diff
@@ -2,4 +2,4 @@
 #
 # Ensure this file is checked in to source control!
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'https://github.com/wordpress-mobile/release-toolkit', tag: '0.13.0'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 1.3.1'
```

Comment:
Side note: you might want to update to `~> 1.4` (and _not_ `~> 1.4.0`, btw) in `develop` at some point during next sprint 🙂